pkg/controller/datavolume/clone-controller-base.go

```diff
@@ -114,6 +115,10 @@ const (
 	RebindInProgress = "RebindInProgress"
 	// MessageRebindInProgress is a const for reporting target rebind
 	MessageRebindInProgress = "Rebinding PersistentVolumeClaim for DataVolume %s/%s"
+	// NoPopulator reports CDI populator is not used so we fallback to host-assisted cloning (reason)
+	NoPopulator = "NoPopulator"
+	// NoPopulatorMessage reports CDI populator is not used so we fallback to host-assisted cloning (message)
+	NoPopulatorMessage = "CDI populator is not used"
```

Comment:
I think something like "in tree storage class does not support snapshot/clone" would be better